### ISSUE TITLE
If I can offer a multi-view image, can I

### ISSUE THREAD

**Issue by ApolloRay** [Tencent/Hunyuan3D-1#26]

**Comment by DeiseFreire** [comment]
> _If I can offer a multi-view image, can I get a more beautiful model ?_ 

Yes, providing multi-view images can improve the quality and accuracy of AI-generated 3D models.
